application/views/responses/browseindex_view.php

```diff
@@ -5,61 +5,62 @@
  * @var $this AdminController
  * @var $num_completed_answers string
  * @var $num_total_answers string
- * @var $tokeninfo mixed
- * @var $with_token CDbTableSchema|null
  */
 
 // DO NOT REMOVE This is for automated testing to validate we see that page
-echo viewHelper::getViewTestTag('surveyResponsesIndex');
+echo viewHelper::getViewTestTag('surveyResponsesBrowse');
 
 ?>
-<div class='side-body <?php echo getSideBodyClass(true); ?>'>
-    <h3><?php eT("Response summary"); ?></h3>
-    <div class="row">
-        <div class="col-12 content-right">
-            <table class='ls-statisticssummary table'>
-                <tbody>
-                    <tr>
-                        <th><?php eT("Full responses"); ?></th>
-                        <td><?php echo $num_completed_answers; ?></td>
-                    </tr>
-                    <tr>
-                        <th><?php eT("Incomplete responses"); ?></th>
-                        <td><?php echo ($num_total_answers - $num_completed_answers); ?></td>
-                    </tr>
-                    <tr class="ls-statisticssummary__sum">
-                        <th><?php eT("Total responses"); ?></th>
-                        <td><?php echo $num_total_answers; ?></td>
-                    </tr>
-                </tbody>
-            </table>
-        </div>
+<div class='side-body survey-response-page <?php echo getSideBodyClass(true); ?>'>
+    <h1> <?= gT("Survey responses") ?> </h1>
+    <div class="mt-4">
+        <?php echo $this->renderPartial(
+            'partial/responseSummary',
+            [
+                'num_completed_answers' => $num_completed_answers,
+                'num_total_answers' => $num_total_answers,
+                'num_total_answers' => $num_total_answers,
```

Comment:
Here is an error probably due to copy paste - you are missing incomplete responses.